js/widgets/page.js

```diff
@@ -46,6 +46,7 @@ $.widget( "mobile.page", {
 
 		// Deprecated in 1.4 remove in 1.5
 		contentTheme: null,
+		enhanceWithin: true,
```

Comment:
Not crazy about this option name while its consistent with the current method the current method will be deprecated when this land in favor of the enhance method from the new enhancer module. I'm actually not sure what I would like to see for an option name here to be honest though.